Main branch fails to build with `build:lib`: error TS1005

### Please describe the task that needs to be done

```
Error: node_modules/parse5/dist/index.d.ts(1,15): error TS1005: ',' expected.
Error: node_modules/parse5/dist/index.d.ts(4,15): error TS1005: ',' expected.
Error: node_modules/parse5/dist/index.d.ts(6,15): error TS1005: ',' expected.
Error: node_modules/parse5/dist/index.d.ts(7,42): error TS1005: ',' expected.
Error: node_modules/parse5/dist/index.d.ts(8,34): error TS1005: ',' expected.
Error: node_modules/parse5/dist/index.d.ts(16,26): error TS1005: ',' expected.
Error: node_modules/parse5/dist/index.d.ts(16,64): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(1,41): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(2,33): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(4,20): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(7,[15](https://github.com/KaotoIO/kaoto-ui/actions/runs/3948618722/jobs/6758826816#step:6:16)): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(7,27): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(7,46): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(7,67): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(7,82): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(7,101): error TS1005: ',' expected.
Error: node_modules/parse5/dist/parser/index.d.ts(7,1[16](https://github.com/KaotoIO/kaoto-ui/actions/runs/3948618722/jobs/6758826816#step:6:17)): error TS1005: ',' expected.
Error: node_modules/parse5/dist/serializer/index.d.ts(2,15): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/index.d.ts(2,15): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/index.d.ts(2,36): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/index.d.ts(2,55): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/index.d.ts(2,70): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/index.d.ts(2,85): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/index.d.ts(3,15): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/preprocessor.d.ts(1,[20](https://github.com/KaotoIO/kaoto-ui/actions/runs/3948618722/jobs/6758826816#step:6:21)): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tokenizer/preprocessor.d.ts(1,38): error TS1005: ',' expected.
Error: node_modules/parse5/dist/tree-adapters/default.d.ts(1,[30](https://github.com/KaotoIO/kaoto-ui/actions/runs/3948618722/jobs/6758826816#step:6:31)): error TS1005: ',' expected.
```

--- a/package.json
+++ b/package.json
@@ -119,7 +119,7 @@
     "tsc-watch": "^6.0.0",
     "tsconfig-paths-webpack-plugin": "4.0.0",
     "tslib": "^2.4.1",
-    "typescript": "4.4.4",
+    "typescript": "^4.9.4",
     "url-loader": "4.1.1",
     "web-worker": "^1.2.0",
     "webpack": "5.75.0",

